tests/generation/test_generate.py

```diff
@@ -160,3 +195,8 @@ def test_general_seq2seq_generation(export_seq2seq_id, export_seq2seq_model_clas
     model = export_seq2seq_model_class.from_pretrained(export_seq2seq_id)
     tokenizer = AutoTokenizer.from_pretrained(export_seq2seq_id)
     _test_model_generation_trn(model, tokenizer, 1, 10, **gen_kwargs)
+
+
+# Mandatory for multiprocessing tests eg. tensor parallel tracing
```

Comment:
Can you elaborate a little bit ? What happens if you don't add this ?

Reply:
Added more.